lib/agent/providers/geo/index.js

```diff
@@ -1,44 +1,35 @@
-"use strict";
+const strategies = require('./strategies');
+const hooks = require('../../hooks');
+const logger = require('../../common').logger.prefix('geo');
 
-var strategies = require('./strategies'),
-    hooks      = require('./../../hooks'),
-    os_name    = process.platform.replace('win32', 'windows').replace('darwin', 'mac'),
-    logger     = require('../../common').logger.prefix('geo');
+const strategiesList = ['native', 'wifi', 'geoip'];
```

Comment:
by bye geoip?

Reply:
Removing geoip causes some troubles, so it may be best to remove it later.